int32 freq
---
tagloc/Tag current
